project/src/main/java/com/google/sps/data/VolunteeringOpportunity.java

```diff
@@ -0,0 +1,117 @@
+package com.google.sps.data;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public final class VolunteeringOpportunity {
+  private String name;
+  private int numSpotsLeft;
+  private Set<String> requiredSkills;
+  private Set<String> volunteers;
```

Comment:
Also should this be private Set<User> volunteers?

Reply:
Waiting for User class to be merged to master.